### ISSUE TITLE
Consider updating Apache configuration suggestions

### ISSUE THREAD

**Issue by alexfornuto** [linode/docs#1317]
1) It's standard in Linode's LAMP and apache docs to tell readers to turn `KeepAlive` off. This made sense when all servers had 8 cores and memory was the limiting factor.

     Now that the minimum package has one core and 1G of RAM, consider removing this recommendation, or at least expanding on the pros and cons of both options.

2) Current suggested settings in the LAMP on Debian 8 guide are for a Linode 2GB, which was the smallest plan when it was updated last (by me). Consider revising these suggestions across the board for 1G plans.

3) Y'all still haven't made a Debian 9 version of the LAMP guide? Still doing the bounty?

**Comment by kcraig34** [comment]
Thanks, @alexfornuto for the insight and rationale.
Bounty was resurrected mid-summer.
You offering to do the update?

**Comment by alexfornuto** [comment]
> Bounty was resurrected mid-summer.
Incomplete update is incomplete. Restructured to what?
>You offering to do the update?
That depends on the results of the first line of inquiry.

**Comment by alexfornuto** [comment]
@GuessWhoSamFoo the website link in your GH profile is missing a t in the protocol. Also, it's pointing to an nginx default page.

**Comment by kcraig34** [comment]
@EdwardAngert -- could you pick up the communication sequence with Alex, please. I'm not sure about his "Incomplete update is incomplete. Restructured to what?" comment.
Can you follow up whenever you get a chance? Thanks.

**Comment by EdwardAngert** [comment]
@alexfornuto we've got a number of LAMP guides slated to update, and we've added your second point to the list of considerations.

As for number 1, good point - that's being deployed in our guides [as we update](https://github.com/linode/docs/blame/master/docs/web-servers/lamp/install-lamp-stack-on-ubuntu-16-04.md#L62)

3rd point: new LAMP for Debian guide is on the list. Once we get to it, we'll keep you in mind if we're low on bandwidth.

For now though, closing
